assets/aircraft/js32.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace BAe-3200 Jetstream Super 31",
```

Comment:
@indianbhaji Apparently JS32 already existed, but with a capitalized filename. That has been changed in another pull request and therefore there is a conflict now. I see that some things differ between the files, for example the full aircraft name. Which one do you think is best? Mine or the old one?

Reply:
Alright, I've changed to using the one from develop.